### ISSUE TITLE
Consistent pipe character

### ISSUE THREAD

**Issue by Doodler8888** [smilhey/ed-cmd.nvim#7]
The plugin uses a pipe character in the insert mode, but only if there is a character after it.

For example if i'm on the last character in the normal state:

![image](https://github.com/user-attachments/assets/9bf65a80-5eb8-4f04-9c66-38bb2e5cd162)

And then press 'a', it wont use to the pipe character:

![image](https://github.com/user-attachments/assets/038f64dc-f1f1-46af-8bd3-cdfc5711523e)

But if there any other character after, like i can add a whitespace after the "o" character, then there will be a pipe character in the insert state:

![image](https://github.com/user-attachments/assets/2e199661-6a29-46df-aaac-a7afb031352d)

Would be nice if there is a way to fix it somehow.

Btw, thanks for the plugin :)

**Comment by smilhey** [comment]
Is that different than the behavior you observe without the plugin ?

**Comment by Doodler8888** [comment]
Without the plugin the pointer is always a block.

**Comment by smilhey** [comment]
When I move the cursor, without the plugin I observe the same behavior, a block when it is on the last character and a pipe character otherwise (moving around with arrows),

**Comment by Doodler8888** [comment]
Correct, i've never used arrow keys for the cmd line, so i missed on that. Which means it's something on the part of neovim?

**Comment by smilhey** [comment]
Yes, you should be able to change that using the guicursor option !
